### DISCUSSION TITLE
SSG build failed! need for help

### DISCUSSION THREAD

**Discussion by Eacolt** [vikejs/vike#982] [Discussion]
`import ssr from 'vite-plugin-ssr/plugin';
 
 ...
plugins:[
...

 ssr({prerender: true}),
]

`
when npm run build, there are wrong information:
build.rollupOptions.input is an HTML entry which is forbidden when using vite-plugin-ssr, instead follow https://vite-plugin-ssr.com/add  ... 

, What can I do? :

error during build:
Error: [vite-plugin-ssr][Wrong Usage] The entry /Users/eacolt/Documents/works/index.html of config build.rollupOptions.input is an HTML entry which is forbidden when using vite-plugin-ssr, instead follow https://vite-plugin-ssr.com/add
    at assertRollupInput ...

**Comment by brillout** [comment]
What is it you want to achieve? Is there a reason you can't apply the steps described at https://vite-plugin-ssr.com/add?

**Reply by Eacolt** [reply]
I want to use pre-rendering , SSG : [https://vite-plugin-ssr.com/pre-rendering](url)
but when I build project, its get wrong

**Reply by brillout** [reply]
I don't understand the problem. AFAICT: follow the instructions and you should be fine.
